### ISSUE TITLE
`traci.vehicle.couldChangeLane` lagging one time step behind

### ISSUE THREAD

**Issue by c433li** [DLR-TS/sumo#3758]
After a lane change via `traci.vehicle.changeLane(veh_id, target_lane_index, time_step_in_ms)`, the lane index is correctly updated in the next step, but the result of `traci.vehicle.couldChangeLane()` still retains the old value until 2 steps later.

In addition, if I make vehicle do yet another (incorrect) lane change through traci according to the outdated `couldChangeLane() value`, the graphics of sumo-gui will hang, and I'll have to kill the process through command line.

**Comment by behrisch** [comment]
I could not reproduce it here. Which sumo version are you using?

**Comment by c433li** [comment]
I tested it on the master branch today, and was able to reproduce it.
![couldchangelane](https://user-images.githubusercontent.com/21252520/35417684-821f5c9c-01fc-11e8-9d84-2f0b4077cb2c.png)
getlaneIndex() returns the same result but couldChangeLane() results change due to the lag

**Comment by behrisch** [comment]
Yes, sorry, I misread the outputs. But I could not see a hanging GUI, did this part disappear on your side as well?

**Comment by c433li** [comment]
No, the GUI hang is still there.
![couldchangelane](https://user-images.githubusercontent.com/21252520/35450673-b29d63e0-028e-11e8-9680-577eea5444df.png)
The program has exited in the command line, but the GUI just got stuck there and cannot be closed.
